File: 1266-minimum-time-visiting-all-points/1266-minimum-time-visiting-all-points.py
class Solution:
    def minTimeToVisitAllPoints(self, points: List[List[int]]) -> int:
        total = 0
        for i in range(len(points)-1): # 0 1 
            a = abs(points[i+1][0] - points[i][0]) 
            b = abs(points[i+1][1] - points[i][1]) 
            total += max(a,b) 
        return total

File: 1266-minimum-time-visiting-all-points/1266-minimum-time-visiting-all-points.java
class Solution {
    public int minTimeToVisitAllPoints(int[][] points) {
        int a = 0;
        int b = 0;
        int total = 0;
        for (int i = 0; i < points.length-1;i++){
            a = Math.abs(points[i+1][0] - points[i][0]);
            b = Math.abs(points[i+1][1] - points[i][1]);
            total +=  Math.max(a,b);
        }
        return total;        
    }
}